data/data.json

```diff
@@ -15,7 +15,8 @@
       "alpha_3": "afg"
     },
     "tld": ".af",
-    "constitutional_form": "republic"
+    "constitutional_form": "republic",
+    "language_codes": ["ps-AF", "fa-AF"]
```

Comment:
Can we rearrange `language_codes {Array}` elements to be sorted in alphabetical order? 
(same for lines: 19, 323, 399, 532, 570 628, 647, 1065, 1464, 1540, 1559, 1673, 1749, 1863, 1958, 1996, 2091, 2110, 2338, 2357, 2528, 2642, 2752, 2824, 2932, 3058, 3094, 3256, 3274, 3328, 3400, 3436, 3454, 3544,  3580,